packages/protocol/scripts/foundry/create_and_migrate_anvil_l2_devchain.sh

```diff
@@ -26,9 +22,119 @@ CELO_UNRELEASED_TREASURY_ADDRESS=$(
     --rpc-url $ANVIL_RPC_URL
 )
 
+RESERVE_ADDRESS=$(
```

Comment:
Not sure if this is what you meant, but I think this code is already implemented in Solidity here https://github.com/celo-org/celo-monorepo/pull/11323/files

Either way, I'd do the minimal work to get this to work in this PR

Reply:
This currenlty works as is.

Had to do this here, because the balance gets overriden when running the L2 script. this only becaumes obvious during testing.

see comment below (L146)